src/main/java/com/github/onsdigital/babbage/response/BabbageStringResponse.java

```diff
@@ -21,14 +21,8 @@ public BabbageStringResponse(String data, String mimeType) {
         this.data = data;
     }
 
-    public BabbageStringResponse(String data, String mimeType, String charEncoding) {
-        this(data, mimeType);
-        setCharEncoding(charEncoding);
-    }
-
```

Comment:
Why this removal?

Reply:
I only removed it because it did not appear to have any usages. However, I've re-added it now to be on the safe side.